server/service/integration_mdm_profiles_test.go

```diff
@@ -1820,6 +1820,13 @@ func (s *integrationMDMTestSuite) TestBatchSetMDMAppleProfiles() {
 	t := s.T()
 	ctx := context.Background()
 
+	var b strings.Builder
+	b.Grow(1000000)
+	for i := 0; i < 1000000; i++ {
+		b.WriteByte('a')
+	}
+	bigString := b.String()
```

Comment:
Nit: you can use `strings.Repeat` to get a big string in a simpler call.

Reply:
(and looks like this wouldn't hit the limit)

Reply:
Copypasta strikes again! I'd seen some debate online about the efficiency of strings.Builder vs
strings.Repeat, but I think using the simpler code makes more sense.

Reply:
Also odd; these tests passed on my machine. I would have expected them to fail? Not sure why they passed...

Reply:
> I'd seen some debate online about the efficiency of strings.Builder vs
strings.Repeat

It uses `strings.Builder` internally, and takes some optimization steps too: https://cs.opensource.google/go/go/+/refs/tags/go1.22.4:src/strings/strings.go;l=537

> these tests passed on my machine. I would have expected them to fail? 

Yes, it's worth taking a minute to understand why that wasn't the case, maybe it will tell us we've missed something.